### DISCUSSION TITLE
Make people be able to vote wether to ban someone

### DISCUSSION THREAD

**Discussion by DWAA1660** [commandgeek/geeksmp#80] [Ideas]
### What and How?

Some people just want to cause mischief in the server ad stretch everything to its limits and nobody wants them on since this is *Kinda* a smp if people vote to remove them either by a large majority of players and staff or unanimous, remove them from the smp.

### Why?

They just kind of ruin the server and makes it less fun for everyone else

### Are there alternatives?

Just to keep them undead and make everyone not wanna play becuz the annoyance

### Checks

- [X] I have used the search at least once to check if my idea has already been suggested and perhaps already implemented.

**Comment by Lianecx** [comment]
Discrimination

**Comment by Lianecx** [comment]
i am definitely not a fan of literally Banning people just because they dont like them

**Reply by DWAA1660** [reply]
if its a unanimus decision do you really think they should be on smp when they just bend rules

**Reply by srnyx** [reply]
> if its a unanimus decision do you really think they should be on smp when they just bend rules

Well, it isn't really fair to them. It's basically just bullying people out of the server.
I've experienced this first-hand where I was vote-banned. Some people that voted had no idea who I was, what I did, etc...
It can create a very toxic community by turning members on each other. *The server I was banned from was shutdown later on lmao*

Just gonna deny this now actually.
